cg/constants/delivery.py

```diff
@@ -206,6 +210,10 @@
         "case_tags": RNAFUSION_ANALYSIS_CASE_TAGS,
         "sample_tags": RNAFUSION_ANALYSIS_SAMPLE_TAGS,
     },
+    Workflow.TAXPROFILER: {
+        "case_tags": TAXPROFILER_ANALYSIS_CASE_TAGS,
```

Comment:
If they are the same, I suggest defining these tags directly with `HermesFileTag.CLINICAL_DELIVERY`. If we have the need to change it we can do it when it is needed.

Reply:
Ok, I see this change is out of scope as it would involve other NF pipelines. Disregard but keep in mind for the future